react/components/Image/index.js

```diff
@@ -45,6 +53,10 @@ Image.schema = {
         'ui:widget': 'image-uploader',
       },
     },
+    href: {
+      type: 'string',
+      title: 'admin/editor.image.href.title',
```

Comment:
Use contentSchema instead

Reply:
Use this native type
https://github.com/vtex-apps/native-types/blob/master/pages/contentSchemas.json#L52-L74

Reply:
![image](https://user-images.githubusercontent.com/18701182/61652740-096dc380-ac8f-11e9-8a63-81b8fac70a49.png)
Cê acha que esses campos fazem sentido para o redirecionamento da imagem? Pode me explicar melhor o que cada um faz?